core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -104,6 +104,11 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     consumer.subscribe(tp)
     consumer.seek(tp, 0)
 
+    // wait until all the followers have synced the last HW with leader
```

Comment:
Yes, I think it's better to track this work around, as we may have to remove this in the future.